tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/InstrumentationTests.cs

```diff
@@ -65,6 +65,40 @@ public async Task DoesNotInstrumentDotnetBuild()
             Task<string> RunDotnet(string arguments) => RunDotnetCommand(workingDir, agent, arguments);
         }
 
+        [SkippableTheory]
+        [InlineData("VSdbG")] // list has vsdbg and vsdbg.exe
+        [InlineData("dd-TRACE")] // list has dd-trace and dd-trace.exe
+        [Trait("RunOnWindows", "True")]
+        public async Task DoesNotInstrumentExcludedNames(string excludedProcess)
```

Comment:
It would be nice to have either a test-case or a process we _should_ instrument, or at the end of the test to rename the executable and see if it gets instrumented. Just a safe guard in case we break the test at some point in the future and it keeps passing just because instrumentation isn't properly setup.